src/sentry/static/sentry/app/components/modals/helpSearchModal.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import {ClassNames, css} from '@emotion/core';
+
+import {t} from 'app/locale';
+import theme from 'app/utils/theme';
+import {ModalRenderProps} from 'app/actionCreators/modal';
+import HelpSearch from 'app/components/helpSearch';
+import Hook from 'app/components/hook';
+import {Organization} from 'app/types';
+
+type Props = ModalRenderProps & {organization: Organization};
+
+const HelpSearchModal = ({Body, closeModal, organization, ...props}: Props) => (
+  <Body>
+    <ClassNames>
+      {({css: injectedCss}) => (
+        <HelpSearch
+          {...props}
+          entryPoint="sidebar_help"
+          dropdownStyle={injectedCss`
+                width: 100%;
+                border: transparent;
+                border-top-left-radius: 0;
+                border-top-right-radius: 0;
+                position: initial;
+                box-shadow: none;
+                border-top: 1px solid ${theme.borderLight};
+              `}
+          renderInput={({getInputProps}) => (
+            <InputWrapper>
+              <Input
+                autoFocus
+                {...getInputProps({
+                  type: 'text',
+                  placeholder: t('Search for documentation, FAQs, blog posts...'),
+                })}
+              />
+            </InputWrapper>
+          )}
+          resultFooter={<Hook name="help-modal:footer" {...{organization, closeModal}} />}
+        />
+      )}
+    </ClassNames>
+  </Body>
+);
+
+const InputWrapper = styled('div')`
+  padding: 2px;
+`;
+
+const Input = styled('input')`
+  width: 100%;
+  padding: 8px;
+  border: none;
+  border-radius: 8px;
```

Comment:
please, does it have to be 8px? because maybe you could use the border-radius of the theme.

Reply:
If we change this we should change this in a few different places, since all of the omni-search modals use this border radius.

Ideally we should just reuse these styles across our omni searches.